spec/triggers/README.md

```diff
@@ -1,11 +1,107 @@
-# KEDA Triggers
+# Trigger Specification
 
-This section describes all the specifications for all supported triggers in KEDA.
+This specification describes the `trigger` section of the `ScaledObject` used to define how what triggers KEDA should use to scale your application.
+
+<details>
+  <summary><b>Table of Contents</b></summary>
+
+- [Metadata](#metadata)
+- [Authentication](#authentication)
+    - [Environment variable(s)](#environment-variables)
+    - [Secret(s)](#secrets)
+    - [Azure Pod Identity](#azure-pod-identity)
+- [Supported Trigger Typess](#supported-trigger-types)
+</details>
+
+[`types.go`](./../pkg/apis/keda/v1alpha1/types.go)
+
+```yaml
+type: {trigger-type} # Required.
+metadata:
+    # {list of properties to configure a trigger}
+authentication:
+    azurePodIdentity: true # Optional. Default: false
```

Comment:
Suggestion by @Aarthisk 	is to consider renaming this to `podIdentity`, `nativeIdentity` or something similar given a scaler will be specific for a given cloud provider anyhow.

Reply:
Another approach is the following:
```yaml
podIdentity:
  provider: none | azure | gcp | spiffe # Optional. Default: none
```

Reply:
I like this with the provider- if we detect it in the helm install we can get it as a config parameter for the scale controller itself

Reply:
Agree, updated the proposal.